src/deploy/functions/params.ts

```diff
@@ -232,7 +232,10 @@ export async function resolveParams(
 ): Promise<Record<string, ParamValue>> {
   const paramValues: Record<string, ParamValue> = {};
 
-  for (const param of params.filter((param) => userEnvs.hasOwnProperty(param.param))) {
+  const [resolved, outstanding] = partition(params, (param) => {
```

Comment:
nit: my instinct is that "resolved" might not be the right variable name here given that them already being in the .env files is more like "doesn't need to be resolved"--maybe something like `fromEnvs` and `toResolve`, off the top of my head?